include/okapi/api/units/RQuantity.hpp

```diff
@@ -192,6 +193,157 @@ constexpr bool operator>(const RQuantity<M, L, T, A> &lhs, const RQuantity<M, L,
+constexpr RQuantity<std::ratio_divide<M, std::ratio<3>>,
+                    std::ratio_divide<L, std::ratio<3>>,
+                    std::ratio_divide<T, std::ratio<3>>,
+                    std::ratio_divide<A, std::ratio<3>>>
+cbrt(const RQuantity<M, L, T, A> &rhs) {
+  return RQuantity<std::ratio_divide<M, std::ratio<3>>,
+                   std::ratio_divide<L, std::ratio<3>>,
+                   std::ratio_divide<T, std::ratio<3>>,
+                   std::ratio_divide<A, std::ratio<3>>>(std::cbrt(rhs.getValue()));
+}
+
+template <typename M, typename L, typename T, typename A>
+constexpr RQuantity<M, L, T, A> hypot(const RQuantity<M, L, T, A> &lhs,
+                                      const RQuantity<M, L, T, A> &rhs) {
+  return RQuantity<M, L, T, A>(std::hypot(lhs.getValue(), rhs.getValue()));
+}
+
+template <typename M, typename L, typename T, typename A>
+constexpr RQuantity<M, L, T, A> max(const RQuantity<M, L, T, A> &lhs,
+                                    const RQuantity<M, L, T, A> &rhs) {
+  return RQuantity<M, L, T, A>(std::max(lhs.getValue(), rhs.getValue()));
+}
+
+template <typename M, typename L, typename T, typename A>
+constexpr RQuantity<M, L, T, A> min(const RQuantity<M, L, T, A> &lhs,
+                                    const RQuantity<M, L, T, A> &rhs) {
+  return RQuantity<M, L, T, A>(std::min(lhs.getValue(), rhs.getValue()));
+}
+
+template <typename M, typename L, typename T, typename A>
+constexpr RQuantity<M, L, T, A> mod(const RQuantity<M, L, T, A> &lhs,
+                                    const RQuantity<M, L, T, A> &rhs) {
+  return RQuantity<M, L, T, A>(std::fmod(lhs.getValue(), rhs.getValue()));
+}
+
+template <typename M1,
+          typename L1,
+          typename T1,
+          typename A1,
+          typename M2,
+          typename L2,
+          typename T2,
+          typename A2>
+constexpr RQuantity<M1, L1, T1, A1> copysign(const RQuantity<M1, L1, T1, A1> &lhs,
+                                             const RQuantity<M2, L2, T2, A2> &rhs) {
+  return RQuantity<M1, L1, T1, A1>(std::copysign(lhs.getValue(), rhs.getValue()));
+}
+
+// Common trig functions:
+// ------------------------------
+
+template <typename M, typename L, typename T, typename A>
+constexpr RQuantity<M, L, T, std::ratio_subtract<A, std::ratio<1>>>
+sin(const RQuantity<M, L, T, A> &rhs) {
+  return RQuantity<M, L, T, std::ratio_subtract<A, std::ratio<1>>>(std::sin(rhs.getValue()));
+}
+
+template <typename M, typename L, typename T, typename A>
+constexpr RQuantity<M, L, T, std::ratio_subtract<A, std::ratio<1>>>
+cos(const RQuantity<M, L, T, A> &rhs) {
+  return RQuantity<M, L, T, std::ratio_subtract<A, std::ratio<1>>>(std::cos(rhs.getValue()));
+}
+
+template <typename M, typename L, typename T, typename A>
+constexpr RQuantity<M, L, T, std::ratio_subtract<A, std::ratio<1>>>
+tan(const RQuantity<M, L, T, A> &rhs) {
+  return RQuantity<M, L, T, std::ratio_subtract<A, std::ratio<1>>>(std::tan(rhs.getValue()));
+}
```

Comment:
I don't think we should keep the units from the operand. These trig functions all return a unitless value so I think our API should reflect that.